frontend/src/components/my-grant-profile-page-handler/grant-table-info.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Button,
+  Grid,
+  GridColumn,
+  Link,
+  Spacer,
+  Text,
+  VerticalStack,
+} from "@python-italia/pycon-styleguide";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentLanguage } from "~/locale/context";
+import { DeadlineStatus, Status as GrantStatus } from "~/types";
+import type { MyProfileWithGrantQuery } from "~/types";
+
+import { getCountryLabel } from "~/helpers/country-utils";
+import { useCountries } from "~/helpers/use-countries";
+import { createHref } from "../link";
+import { Sidebar } from "./sidebar";
+
+type Props = {
+  grant: MyProfileWithGrantQuery["me"]["grant"];
+};
+const GrantTableInfo = ({ grant }: Props) => {
+  const countries = useCountries();
+  return (
+    <Grid cols={3} gap="small" fullWidth>
+      <div>
```

Comment:
you can also extract those items components in a <TableItem> component so that you don't repeat the same thing